setup.py

```diff
@@ -12,7 +12,7 @@
 
 def get_requirements(reqfile):
     path = os.path.join(HERE, reqfile)
-    deps = set()
+    deps = list()
```

Comment:
Whats the reason for changing from set to list?

Reply:
The base container (python) has been updated. Now 'setup tools' expect get_requirements() function to return a list.
We had to do the same update on auth: https://github.com/dojot/auth/pull/25/files

Reply:
Ok.